cmd/snap/cmd_prepare_image.go

```diff
@@ -97,6 +98,8 @@ For preparing classic images it supports a --classic mode`),
 			"channel": i18n.G("The channel to use"),
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"customize": i18n.G("Image customizations specified as JSON file."),
+			// TRANSLATORS: This should not start with a lowercase letter.
+			"validation": i18n.G("Control whether validations should be ignored or enforced."),
```

Comment:
should this say something about the default?

Reply:
Good point. 
I have taken this directly from the `ubuntu-image`. I have now added that default is `ignored`